src/_gettsim/interface.py

```diff
@@ -0,0 +1,83 @@
+import pandas as pd
+from dags.tree.typing import NestedTargetDict
+
+from _gettsim.config import RESOURCE_DIR, SUPPORTED_GROUPINGS
+from ttsim import (
+    compute_taxes_and_transfers,
+    create_data_tree_from_df,
+    set_up_policy_environment,
+)
+from ttsim.typing import NestedDataDict, NestedInputToSeriesNameDict
+
+
+def quickrun(
+    date: str,
+    df: pd.DataFrame,
+    input_tree_to_column_map: NestedInputToSeriesNameDict,
```

Comment:
```suggestion
    inputs_tree_to_df_columns: NestedInputsPathsToDfColumns,
```
- plural consistent for inputs/targets
- df should appear again
- map seems unnecessary
- Type hint: We'll need another round to make them consistent, anyhow. Not too important. But `SeriesName` is not a concept I'd like to see introduced :wink:.